spec/serializer_spec.rb

```diff
@@ -335,8 +352,10 @@ def new_test
     end
 
     context 'serializer precedence call' do
-      it 'inkokes the instance method instead of the object one' do
+      it 'invokes the instance method instead of the object one' do
         expect(hash[:new_test]).to eq(instance.new_test)
+        expect(hash[:private_test]).to eq(instance.send(:private_test))
```

Comment:
It is better to use hardcoded values for asserting instead of `instance.send(:blah blah blah)`